atomic_reactor/plugins/pre_pull_base_image.py

```diff
@@ -124,7 +126,7 @@ def _get_image_with_digest(self, image):
         new_image = '{}@sha256:{}'.format(image_name, digest)
         return ImageName.parse(new_image)
 
-    def _store_manifest_digest(self, image, use_original_tag):
+    def _store_manifest_digest(self, image: ImageName, use_original_tag: bool) -> None:
```

Comment:
Redundant? Functions return `None` by default...

Reply:
This is really a good question. I think there should be many aspects of the consideration to answer this question. In short, explicit would be better than implicit to describe the behavior of a function. Annotating None here means this method does not return anything.